mergify_engine/actions/queue.py

```diff
@@ -258,10 +259,14 @@ async def run(
             if await self._should_be_queued(ctxt, q):
                 await q.add_pull(ctxt, typing.cast(queue.PullQueueConfig, self.config))
                 try:
-                    if await self._should_be_merged(ctxt, rule, q):
+                    qf = await freeze.QueueFreeze.get(
+                        ctxt.repository, self.config["name"]
+                    )
+                    if await self._should_be_merged(ctxt, q, qf):
                         result = await self._merge(ctxt, rule, q, merge_bot_account)
                     else:
-                        result = await self.get_queue_status(ctxt, rule, q)
+                        result = await self.get_queue_status(ctxt, rule, q, qf)
+
```

Comment:
I see no test on this part in the functional tests.

Reply:
`assert (
            check["output"]["title"]
            == 'The queue "default" is currently frozen, for the following reason: "test freeze reason"'
        )`
there is this part in test_create_queue_freeze(), it asserts the PR status in the queue

Reply:
I missed it thanks!